cogs/utils/drawdota.py

```diff
@@ -721,6 +722,44 @@ def get_datetime_cell(match, region_data):
 		TextCell(str_time, font_size=18, horizontal_align="center")
 	)
 
+async def draw_meta_table(sorted_heroes, heroes): 
+	"""Takes a list of [heroes+p/b%+wr%] and draws a nice little discord-friendly table"""
+	
+	border_size = 10
+	grey_color = "#BBBBBB"
+	table = Table(background=discord_color2)
+	#Header
+	headers=[
+		TextCell("Hero"), 
+		TextCell("", padding=0), 
+		TextCell("Win %"), 
+		TextCell("Pick/Ban %")
+	]
+
+	table.add_row(headers)
+	for cell in table.rows[0]: 
+		cell.background = discord_color1
+
+	table.add_row([ColorCell(color=discord_color1, height=6) for i in range(len(headers))])
```

Comment:
Is there a reason to do this here instead of adding it via `padding=6` to the header cells? That'd also have the benifit of adding a bit of extra space at the top.

Reply:
Definitely no reason in particular. I think the only reason I did that was because I was following what we were doing in `draw_matches_table`, but seems like the results are the same. I just pushed it and removed some stuff that wasn't being used.